.github/workflows/ci.yml

```
@@ -7,11 +7,21 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        gemfile: [rails_4.0.0.gemfile, rails_5.0.0.gemfile, rails_5.2.3.gemfile, rails_6.0.0.gemfile, rails_master.gemfile]
-        ruby_version: [2.4.x, 2.5.x, 2.6.x, 2.7.x]
+        gemfile: [rails_4.0.0.gemfile, rails_5.0.0.gemfile, rails_5.2.3.gemfile, rails_6.0.0.gemfile, rails_7.0.0.gemfile, rails_master.gemfile]
+        ruby_version: [2.4, 2.5, 2.6, 2.7, 3.0]
         exclude:
           - gemfile: rails_master.gemfile
             ruby_version: 2.4.x
+          - gemfile: rails_master.gemfile
+            ruby_version: 2.5.x
+          - gemfile: rails_master.gemfile
+            ruby_version: 2.6.x
+          - gemfile: rails_7.0.0.gemfile
+            ruby_version: 2.4.x
+          - gemfile: rails_7.0.0.gemfile
+            ruby_version: 2.5.x
+          - gemfile: rails_7.0.0.gemfile
+            ruby_version: 2.6.x
           - gemfile: rails_6.0.0.gemfile
             ruby_version: 2.4.x
```

I think this is needed as well

```suggestion
          - gemfile: rails_master.gemfile
            ruby_version: 2.4
          - gemfile: rails_master.gemfile
            ruby_version: 2.5
          - gemfile: rails_master.gemfile
            ruby_version: 2.6
          - gemfile: rails_7.0.0.gemfile
            ruby_version: 2.4
          - gemfile: rails_7.0.0.gemfile
            ruby_version: 2.5
          - gemfile: rails_7.0.0.gemfile
            ruby_version: 2.6
          - gemfile: rails_6.0.0.gemfile
            ruby_version: 2.4
```